Terminal.Gui/Application/Application.cs

```diff
@@ -975,7 +975,7 @@ public static void RunIteration (ref RunState state, ref bool firstIteration)
 
         if (state.Toplevel.NeedsDisplay || state.Toplevel.SubViewNeedsDisplay || state.Toplevel.LayoutNeeded || OverlappedChildNeedsDisplay ())
         {
-            state.Toplevel.SetNeedsDisplay();
+            state.Toplevel.SetNeedsDisplay ();
```

Comment:
I have no idea why the diff is showing this was added. This line is in v2_develop.

Reply:
You can leave it like this because it is within the required format parameters.